v20.1/cockroach-demo.md

```diff
@@ -8,30 +8,35 @@ The `cockroach demo` [command](cockroach-commands.html) starts a temporary, in-m
 
 The in-memory cluster persists only as long as the SQL shell is open. As soon as the shell is exited, the cluster and all its data are permanently destroyed. This command is therefore recommended only as an easy way to experiment with the CockroachDB SQL dialect.
 
-
 Each instance of `cockroach demo` loads a temporary [enterprise license](https://www.cockroachlabs.com/get-cockroachdb) that expires after an hour.
 
 ## Synopsis
 
-Start an interactive SQL shell:
+Start an in-memory cluster and open an interactive SQL shell:
 
 ~~~ shell
 $ cockroach demo <flags>
 ~~~
 
-Load a sample dataset and start an interactive SQL shell:
+Start an in-memory cluster with a preloaded dataset and open an interactive SQL shell:
 
 ~~~ shell
 $ cockroach demo <dataset> <flags>
 ~~~
 
-Execute SQL from the command line:
+Start an in-memory cluster in secure mode and open an interactive SQL shell:
```

Comment:
Is it worth mentioning here that this lets you try out stuff like permissions?

Reply:
Done. @Amruta-Ranade will add an example in a follow-up PR.